examples/movies/server/src/movies/package.scala

```diff
@@ -98,7 +102,14 @@ package object movies {
       implicit val mapper = if (req.pretty) prettyJson else textJson
       val rsp = req.response
       rsp.status = Status.Ok
-      rsp.etag = time
+      //rsp.etag = time
+      //Added
+	  rsp.date = req.lastModificationBefore
```

Comment:
I've already done the formatting. It is in the twitter-server/..../package.scala file.